practice-app/src/todos/Priority.tsx

```diff
@@ -8,16 +8,19 @@ import {
 } from "@/components/ui/select";
 import type { Dispatch } from "react";
 import { Actions } from "./reducer";
+import TodoItem from "./ToDoItem";
 
-export function PrioritySelector(
-  props: Pick<ToDo, "priority" | "id"> & { dispatch: Dispatch<Actions> }
-) {
+export function PrioritySelector(todo: ToDo & { dispatch: Dispatch<Actions> }) {
   return (
     <Select
       onValueChange={(value: ToDo["priority"]) => {
         // TODO: update the priority of the todo item
```

Comment:
@Prec1se the code looks good to me! But, could you remove the `TODO` comment as the work here is done?

Reply:
of course